src/test/java/io/stargate/sgv2/jsonapi/TestConstants.java

```diff
@@ -57,7 +57,7 @@ public final class TestConstants {
           VectorConfig.NOT_ENABLED_CONFIG,
           null,
           CollectionLexicalConfig.configForDisabled(),
-          CollectionRerankingConfig.configForLegacyCollections());
+          CollectionRerankingConfig.configForPreRerankingCollections());
```

Comment:
this is going to change after I merge the multi vector, we stop making these as statics